src/pages/settings/Profile/ProfilePage.js

```diff
@@ -228,11 +206,19 @@ class ProfilePage extends Component {
                         onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
                         onCloseButtonPress={() => Navigation.dismissModal(true)}
                     />
-                    <ScrollView style={styles.flex1} contentContainerStyle={styles.p5}>
+                    <Form
+                        style={[styles.p5]}
```

Comment:
I'm don't think the `Save` button will be bottom docked here. I believe we might need to add `flexGrow1` to make that happen.